api/AnimationEvent.json

```diff
@@ -83,8 +83,8 @@
               "version_added": "4.0"
             },
             {
-              "prefix": "webkit",
-              "version_added": true
+              "version_added": true,
```

Comment:
Why is this change here? It only swaps the order of version_added and prefix?

Reply:
My guess is that it's trying to mirror the order of keys from Chrome?  That would explain the various other instances of reordering as well.  (Probably some Node.js thing if I had to guess, I'll certainly have to keep it in mind!)

Reply:
Probably, yeah. I think we should try to have consistent ordering or have the mirroring script ignore this. It creates unnecessary noise when we do script based mass updates.